pinot-tools/src/main/resources/examples/batch/dimBaseballTeams/dimBaseballTeams_offline_table_config.json

```diff
@@ -0,0 +1,23 @@
+{
+  "tableName": "dimBaseballTeams",
+  "tableType": "OFFLINE",
+  "isDimTable": "true",
```

Comment:
I think we either have `tableType`:`DIMENSION` or `"isDimTable": "true"`, to be consistent with your `TableDataManagerType`